.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -20,16 +20,10 @@ A clear and concise description of what you expected to happen.
 **Screenshots**
 If applicable, add screenshots to help explain your problem.
 
-**Desktop (please complete the following information):**
- - OS: [e.g. iOS]
- - Browser [e.g. chrome, safari]
- - Version [e.g. 22]
-
-**Smartphone (please complete the following information):**
- - Device: [e.g. iPhone6]
- - OS: [e.g. iOS8.1]
- - Browser [e.g. stock browser, safari]
- - Version [e.g. 22]
+**Environment**
+- please paste the result of `npx @brainhubeu/envinfo@latest --preset brainhubeu`
```

Comment:
what if a problem exists on mobile? You don't have terminal there, we still need device/OS

Reply:
@humbak 
I updated in https://github.com/brainhubeu/react-carousel/pull/354/commits/f208d3272bcd24d7f08fb99ced1c401f8a4047f1